### PR TITLE
fix: radio checkbox and selectable card visible input

### PR DESCRIPTION
**Pull request by matthprost**
## Type

- Bug

#### What is expected?

To remove visible input in radio / checkbox / selectable card / switch button.

I made the input as same size as svg but with no opacity so when user click on svg he is actually clicking on hidden input. This way we keep accessibility and tick customisation.

I also fixed hovered state of selectable card, when disabled border was still getting primary color.

## Relevant logs and/or screenshots

| Page |   Before   |      After |
| :--- | :--------: | ---------: |
| url  | Screenshot 2022-09-27 at 14 04 47 | Screenshot 2022-09-27 at 14 04 53 |

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@scaleway/ui",
-  "version": "0.194.1",
+  "version": "2.194.1",
```
Comment (chambo-e) [inline package.json:3]: wut ?

**Review by chambo-e — CHANGES_REQUESTED** [review]

File: src/components/Checkbox/index.tsx [hunk 2]
```diff
@@ -56,14 +56,10 @@ const StyledReakitCheckbox = styled(ReakitCheckbox, {
...
-  width: 1px;
-  height: 1px;
-  padding: 0;
-  margin: -1px;
-  overflow: hidden;
-  clip: rect(0, 0, 0, 0);
   white-space: nowrap;
+  height: ${({ size }) => size}px;
+  width: ${({ size }) => size}px;
+  opacity: 0;
```
Comment (chambo-e) [inline src/components/Checkbox/index.tsx:62]: If I remember correctly, opacity 0 on safari and some other browser is considered non present element, so you won't be able to click on the checkbox

**Review by chambo-e — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (matthprost) [reply package.json:3]: my bad

File: src/components/Checkbox/index.tsx (hunk 2 above)
Comment (matthprost) [reply src/components/Checkbox/index.tsx:62]: @chambo-e it should be fine it is the label who's clickable, the input is inside the label

**Review by chambo-e — APPROVED** [review]

**Review by baptisteArno — APPROVED** [review]

**Pull request merged**